docker-rstudio-oss.tf

```diff
@@ -0,0 +1,76 @@
+resource "github_repository" "docker-rstudio-oss" {
+  name        = "docker-rstudio-oss"
+  description = "A JupyterHub container with required extensions and libraries"
+
+  allow_merge_commit     = false
+  delete_branch_on_merge = true
+  auto_init              = true
+  has_issues             = false
+
```

Comment:
there should be a template block here. The readme says to use https://github.com/dwp/dataworks-github-config/blob/master/docker-template-repository.tf.sample as your starting point